.github/workflows/verify-pr-commit.yml

```diff
@@ -143,6 +143,8 @@ jobs:
         run: |
           export BUILT_BIN_FILENAME=frequency; echo "BUILT_BIN_FILENAME=$BUILT_BIN_FILENAME" >> $GITHUB_ENV
           echo "FINAL_BIN_FILENAME=$BUILT_BIN_FILENAME.${{matrix.network}}.${{matrix.spec}}.${{matrix.branch_alias}}" >> $GITHUB_ENV
+      - name: Install Components
+        run: rustup component add rust-src
```

Comment:
We can remove these , `rust-toolchain.toml` has it?

Reply:
Good suggestion. I did try to remove it, but the CI failed and asked for it. I’ll double-check again; perhaps I overlooked something else.

Reply:
Looks like it does not work, maybe you can drop a story to either include it in CI base image or figure out how to get it installed via tool chain file 


For this PR it's good to include

Reply:
I think you are right. I only needed to add it to the Verify Rust Docs. I removed the others.